components/org.wso2.carbon.identity.conditional.auth.functions.user.store/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/store/UserStoreFunctions.java

```diff
@@ -45,30 +48,23 @@ public class UserStoreFunctions implements GetUserWithClaimValues {
 
     private static final Log LOG = LogFactory.getLog(UserStoreFunctions.class);
 
-    public JsAuthenticatedUser getUniqueUserWithClaimValues(Map<String, String> claimMap, Object... parameters)
+    @HostAccess.Export
+    public JsAuthenticatedUser getUniqueUserWithClaimValues(Map<String, String> claimMap,
+                                                            JsBaseAuthenticationContext context, String... parameters)
```

Comment:
This will not break the existing logic as existing logic also requires Context to be there. 
However a new functional interface is added for the new method signature.